sphinxcontrib/redoc.py

```diff
@@ -30,10 +32,24 @@ def render(app):
         # relies on them.
         ctx.setdefault('opts', {})
 
+        # "Embed" mode: the JSON spec is embedded in the page
+        # TODO: only works for local files: add remote files download ?
```

Comment:
Personal opinion: no TODOs in code, they will never be fixed. :) If you think it's important to implement, could you please create a followup issue instead?

Reply:
I agree, I'll open an issue (and maybe do it if I have some time)